src/app/books-list/book/book-item/book-item.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, OnInit} from '@angular/core';
+import {IBook} from "../../../BOOKS";
+import {ActivatedRoute, RouterLink} from "@angular/router";
+import {BooksService} from "../../../books.service";
+import {CurrencyPipe, DecimalPipe} from "@angular/common";
+
+@Component({
+  selector: 'app-book-item',
+  standalone: true,
+  imports: [
+    DecimalPipe,
+    CurrencyPipe,
+    RouterLink
+  ],
+  templateUrl: './book-item.component.html',
+  styleUrl: './book-item.component.scss'
+})
+export class BookItemComponent implements OnInit {
+  bookId!: string;
```

Comment:
to od razu tutaj przypisz `this.route.snapshot.params['id']` powinno przejść

Reply:
czego resolvowala i nie odpisala jak nie zrobila

Reply:
zrobiła przypisanie do `book: IBook | undefined = this.booksService.getBook(this.route.snapshot.params['id'])`